src/utils/applyStyles/applyStyles.js

```diff
@@ -1,11 +1,11 @@
 // @flow
-import type { TBreakpoint, TBreakpointBehavior } from '../const/defaultOptions'
-import type { TProps } from './parsePropName'
-import propAliases from '../const/propAliases'
-import Layout from '../Layout'
-import toDashedString from './toDashedString'
-import createMediaQuery from './createMediaQuery'
-import parsePropName from './parsePropName'
+import type { TBreakpoint, TBreakpointBehavior } from '../../const/defaultOptions'
+import type { TProps }                           from '../parsePropName'
+import propAliases                               from '../../const/propAliases'
+import Layout                                    from '../../Layout'
+import toDashedString                            from '../toDashedString'
+import createMediaQuery                          from '../createMediaQuery'
+import parsePropName                             from '../parsePropName'
```

Comment:
I don't think I prefer this kind of spacing.
Are you using Prettier? I think it will remove this once it formats the file.

I know some things may not be very well aligned for cooperative development, like coding styleguide, or formatting tools, but we can focus to achieve that unification first.

Reply:
My bad did not notice that, my editor does this on save automatically

Reply:
That's a good catch. Do you know how to disable this project-wide?

Reply:
Yep, I did turn this on globally because that's how I personally prefer it but I can switch this on/off on a language or project basis.